src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -26,15 +26,21 @@ export default class BraintreeHostedForm {
         private _braintreeSDKCreator: BraintreeSDKCreator
     ) {}
 
-    async initialize(options: BraintreeFormOptions): Promise<void> {
+    async initialize(options: BraintreeFormOptions): Promise<boolean> {
         this._formOptions = options;
 
         this._type = isBraintreeFormFieldsMap(options.fields) ?
             BraintreeHostedFormType.CreditCard :
             BraintreeHostedFormType.StoredCardVerification;
 
+        const fields = this._mapFieldOptions(options.fields);
+
+        if (isEmpty(fields)) {
+            return Promise.resolve(false);
```

Comment:
Just return `return false` is fine, because the method is marked as `async`, it'd automatically return a `false` wrapped in a `Promise`